app/locales/pt-PT.json

```diff
@@ -0,0 +1,188 @@
+{
+  "Postal code": "Código postal",
+  "Important": "Importante",
+  "Page not found": "Página não encontrada",
+  "We can't find the page or service you're looking for": "Não nos foi encontrar a página ou serviço que procura",
+  "The link you clicked may be broken, or the page no longer exists": "O endereço que abriu pode estar incompleto ou a página pode não existir",
+  "Something went wrong!": "Algo correu mal!",
+  "Something must have gone wrong": "Algo não está a funcionar corretamente",
+  "Send an e-mail to": "Envie um e-mail para",
+  "if you can't get it to work.": "se o problema persistir.",
+  "Before proceeding please note down the following link": "Antes de continuar por favor tome nota do seguinte endereço",
+  "Save the following link somewhere, or bookmark it in your browser.": "Anote o seguinte endereço ou guarde nos favoritos do seu browser.",
+  "That way you can update your report if your health condition changes.": "Desta forma poderá atualizar o seu formulário caso o seu estado de saúde se altere.",
+  "I've noted down the link.": "Já guardei o endereço.",
+  "Proceed": "Proceder",
+  "The link has been copied": "O endereço foi copiado.",
+  "Copy": "Copiar",
+  "Could not copy the link, please copy it yourself": "Não nos foi possível copiar o endereço, por favor copie o endereço.",
+  "Contributors": "Voluntários",
+  "This website is made with help from volunteers. In no particular order:": "Este website foi criado com a ajuda de voluntários. Sem nenhuma ordem particular:",
+  "...and others": "...e outros",
+  "The code for the website can be found": "O código do website pode ser encontrado",
+  "here": "aqui",
+  "About the service": "Sobre o serviço",
+  "In connection with the outbreak of COVID-19, many people have been tested for the virus, but due to lack of resources, the overwhelming majority have not been tested. It is therefore assumed that there are large missing numbers here in connection with the outbreak.": "No seguimento do surto de COVID-19 muitas pessoas foram testadas para o vírus, mas devido à falta de recursos a grande maioria não terá sido testada. Supõe-se, portanto, que exista um grande números de casos ausentes das estatísticas atuais sobre o surto.",
```

Comment:
I'd add commas after "COVID-19" and "virus" (just like in the original version):

"No seguimento do surto de COVID-19, (...) mas devido à falta de recursos, (...)"